include/wx/msw/private/webview_ie.h

```diff
@@ -0,0 +1,217 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        include/wx/msw/private/webview_ie.h
+// Purpose:     wxMSW IE wxWebView backend private classes
+// Author:      Marianne Gagnon
+// Copyright:   (c) 2010 Marianne Gagnon, 2011 Steven Lamerton
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef wxWebViewIE_PRIVATE_H
+#define wxWebViewIE_PRIVATE_H
+
+#include "wx/msw/ole/automtn.h"
+#include "wx/msw/private/comptr.h"
+#include "wx/msw/private/webview_missing.h"
+
+class ClassFactory;
+class wxIEContainer;
+class DocHostUIHandler;
+class wxFindPointers;
+
+class wxWebViewIEImpl
+{
+public:
+    wxWebViewIEImpl(wxWebViewIE* webview);
```

Comment:
Very minor, but just to avoid any potential problems and questions:
```suggestion
    explicit wxWebViewIEImpl(wxWebViewIE* webview);
```

Similarly, I think this class should use `wxDECLARE_NO_COPY_CLASS()`.